src/api/providers/kilocode-openrouter.ts

```diff
@@ -27,10 +27,10 @@ export class KilocodeOpenrouterHandler extends OpenRouterHandler {
 	}
 
 	constructor(options: ApiHandlerOptions) {
-		const baseUri = getKiloBaseUriFromToken(options.kilocodeToken ?? "")
+		const baseUrl = getKiloBaseUriFromToken(options.kilocodeToken ?? "")
```

Comment:
This seems redundant then? Since this is looking at the token to decide whether or not to connect to localhost (if token comes from local server it will, if not it won't).